addons/account_audit_trail/report/audit_trail_report_views.xml

```diff
@@ -27,7 +27,7 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]"/>
+                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
```

Comment:
Should it be done here too ? https://github.com/odoo/odoo/blob/17.0/addons/l10n_in/report/audit_trail_report_views.xml#L28